### PR TITLE
HERITAGE-177-implement-OHOS-homepage-design-changes

### PR DESCRIPTION
**Pull request by JohnHeeryTNA**
Ticket URL: https://national-archives.atlassian.net/browse/HERITAGE-177 

## About these changes

Design changes implemented as per Figma design:
https://www.figma.com/proto/lFiWMlfg7l6BUAsUk0ZhB6/Remixer-concepts?page-id=0%3A1&type=design&node-id=51-260&viewport=1357%2C1457%2C0.75&t=VbmfgSDRy9aPS6Kz-1&scaling=scale-down-width&starting-point-node-id=51%3A260

## How to check these changes

Run locally, check home page 
http://127.0.0.1:8000/

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes
- [x] Ensured that PR includes only commits relevant to the ticket
- [x] Waited for all CI jobs to pass before requesting a review
- [x] Added/updated tests and documentation where relevant

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: templates/home/home_page.html [hunk 1]
```diff
@@ -26,52 +27,148 @@ <h1 class="tna-heading-xl search-hero__heading">Find community-generated history
...
+                                                <img src="/static/images/ohos-homepage-tags.jpg" alt=""  />
+                                            </picture>
+                                        </a>
+
+                                        <div class="tna-card__body">
+                                            <p>Use AI-generated tags to uncover connections to people, places and organisations of the past.</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </li>
```
Comment (TNA-Allan) [inline templates/home/home_page.html:164]: Same as above 2 points for structure, links

File: templates/home/home_page.html [hunk 2]
```diff
@@ -26,52 +27,148 @@ <h1 class="tna-heading-xl search-hero__heading">Find community-generated history
...
+                                                <img src="/static/images/ohos-homepage-maps.jpg" alt=""  />
+                                            </picture>
+                                        </a>
+
+                                        <div class="tna-card__body">
+                                            <p>Find community history for places of interest to you across the UK and beyond.</p>
+                                        </div>
+                                    </div>
+                                </div>
+                            </li>
```
[the remark this answers is not in the document]
Comment (TNA-Allan) [reply templates/home/home_page.html]: I've added the include template and visualisation urls. Ready for your checks and updates.

File: templates/home/home_page.html (hunk 1 above)
Comment (JohnHeeryTNA) [reply templates/home/home_page.html]: I've run locally and changes seem good. The list items (cards) remain within the <ul> structure, so I think we are ok.

**Review by TNA-Allan — COMMENTED** [review]
Looks good. Thanks @JohnHeeryTNA

**Pull request merged**